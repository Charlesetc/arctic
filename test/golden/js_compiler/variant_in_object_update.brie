define main [
        define x (Hi < x = (There "you")>)
        x = (Hi < x = (There "wow") >)
]
