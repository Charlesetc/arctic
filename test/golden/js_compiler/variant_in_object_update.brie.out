const global_functions = {1: {}};
function main() {
	var x = {Hi: [{x: {There: ["you"]}}]};
	return (x = {Hi: [{x: {There: ["wow"]}}]});
}
var test/golden/js_compiler/variant_in_object_update.brie_main = new_closure(1)
console.log(main());